### DISCUSSION TITLE
Is there a way to have multiple Item names in the advanced block

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#16910] [How to]
Currently, when you create an item name in the advanced block when editing fields, you cannot add another, you are limited to one. However, I was wondering if there was a way I could add a second item name field. The current name field consist of modules products and services, but I want one Item Name field to only have the product module, while having another Name Field only have the service module so that when they are printing in a Quote PDF, their prices and tables are displayed separately rather than combined.

**Comment by rskrzypczak** [comment]
I don't know exactly what you're asking. Maybe a screenshot?
However, as I understand you correctly, you want to have two advanced blocks, but this is not possible.
In the next version of the application it will be possible to create groups, although I don't know if it solves your problem.
![image](https://user-images.githubusercontent.com/10221999/219574453-15808320-7202-4240-af58-9feedf9f8ab7.png)

**Reply by bfeddes** [reply]
This would work for what I am asking, if it allows them to be displayed as separate tables when printing out a quote as a PDF. Currently, I am printing a table of all services & products related to a quote with the following: $(inventory : type=table columns=seq,name,qty,unit,price,total, net  href=no)$. This displays both products and services together. However, ideally, I would want two separate tables. One that shows all services, and one that shows all products, rather than having one combined table that displays both. Our customers tend to prefer the services and products in a quote be separate.

**Reply by rskrzypczak** [reply]
Unfortunately, none of the available functions generating the table from the advanced block has the option of narrowing down the entries.
So it would be necessary to add such a possibility, or create a separate textparser function.
